jupyter-extension/src/plugins/mount/__tests__/mount.test.tsx

```diff
@@ -181,7 +181,7 @@ describe('mount plugin', () => {
     );
     expect(plugin.layout.title.caption).toBe('Pachyderm Mount');
     expect(plugin.layout.id).toBe('pachyderm-mount');
-    expect(plugin.layout.orientation).toBe('vertical');
+    // expect(plugin.layout.orientation).toBe('vertical');
```

Comment:
I think you can safely remove this it is just due to the change here https://github.com/pachyderm/pachyderm/pull/9696/files#diff-de40472f6956deb515294eb444c2027f5d273e2da739514ad1b50bdd62708e59L292. The tabs have the orientation now afiak.